tools/packaging/kata-deploy/kata-deploy/base/kata-deploy.yaml

```diff
@@ -38,6 +38,7 @@ spec:
           value: "false"
         - name: CREATE_DEFAULT_RUNTIMECLASS
           value: "false"
+        hostPID: true
         securityContext:
           privileged: true
```

Comment:
Does it still require `privileged`?

Reply:
We still do. Kata-deploy logs report:
```
nsenter: cannot open /proc/1/ns/mnt: Permission denied
```
when privileged is off (but kata-deploy doesn't catch the failure?)